pages/index.vue

```diff
@@ -225,79 +99,286 @@ const parsedNowShowing = computed(() => {
     }
   })
 })
+
+const parsedQuickLinks = computed(() => {
+  if (page.value.ftvaQuickLinks.length === 0)
+    return null
+
+  return page.value.ftvaQuickLinks.map((item) => {
+    return {
+      title: item.titleGeneral,
+      to: item.urlLink,
+      text: item.description,
+      image: item.image[0],
+    }
+  })
+})
+
+const parsedArchiveBlogs = computed(() => {
+  if (page.value.ftvaFeaturedArticlesSection.length === 0)
+    return null
+
+  const obj = page.value.ftvaFeaturedArticlesSection[0]
+  return {
+    sectionTitle: obj.sectionTitle,
+    sectionCta: obj.seeAllText,
+    blogTitle: obj.featuredArticles[0].title,
+    blogUri: obj.featuredArticles[0].uri,
+    blogSummary: obj.featuredArticles[0].ftvaHomepageDescription,
+    image: obj.featuredArticles[0].ftvaImage
+  }
+})
+
+const parsedFeaturedCollections = computed(() => {
+  if (page.value.ftvaFeaturedCollectionsSectionSingle.length === 0)
+    return null
+
+  const collections = page.value.ftvaFeaturedCollectionsSectionSingle[0].featuredCollections.map((item) => {
+    return {
+      title: item.title,
+      to: item.uri,
+      image: item.imageCarousel && item.imageCarousel.length > 0 ? item.imageCarousel[0].image[0] : null,
+    }
+  })
+
+  return {
+    sectionTitle: page.value.ftvaFeaturedCollectionsSectionSingle[0].sectionTitle,
+    sectionSummary: page.value.ftvaFeaturedCollectionsSectionSingle[0].sectionDescription,
+    sectionCta: page.value.ftvaFeaturedCollectionsSectionSingle[0].seeAllText,
+    collections
+  }
+})
+
+const parsedPreservationData = computed(() => {
+  if (page.value.beforeAfterImageCarousel.length === 0)
+    return null
+
+  return {
+    sectionTitle: page.value.sectionTitle,
+    sectionSummary: page.value.richTextSimplified,
+    sectionUri: page.value.ftvaRelatedResources,
+    beforeImage: page.value.beforeAfterImageCarousel[0].beforeImage[0],
+    afterImage: page.value.beforeAfterImageCarousel[0].afterImage[0],
```

Comment:
add safety check here too, we can also pass fallback value null if beforeImage or afterImage value is undefined